python-lib/gluonts_forecasts/model.py

```diff
@@ -220,6 +219,7 @@ def _get_model_parameters_json(self, train_list_dataset):
                 "timeseries_number": timeseries_number,
                 "timeseries_average_length": round(timeseries_total_length / timeseries_number),
                 "model_parameters": self.model_parameters,
+                "mxnet.context": str(self.ctx),
```

Comment:
What's the output of this for cpu? And for gpu?

Reply:
the output is "gpu($GPU_NUMBER)" or "cpu(0)"

Reply:
Perfect